<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Check_Out</class>
 <widget class="QMainWindow" name="Check_Out">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>934</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>901</width>
      <height>851</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget1">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>170</y>
      <width>401</width>
      <height>311</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
   </widget>
   <widget class="QLabel" name="OrderDateLabel">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>150</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TotalPriceLabel">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>480</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="GreetingLabel">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>50</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ImageLabel">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="GoodbyeLabel">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>510</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1340</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
